packages/dd-trace/test/plugins/util/__test__/CODEOWNERS

```
@@ -1,3 +1,3 @@
 # CODEOWNERS for testing purposes
-packages/dd-trace/test/plugins/util/* @datadog-apm-js
+packages/dd-trace/test/plugins/util/* @datadog-dd-trace-js
 packages/dd-trace/test/plugins/util/test.spec.js @datadog-ci-app
```

I'm not sure it matters since this is only a test fixture, but the names you specified don't exist. These do:
```suggestion
packages/dd-trace/test/plugins/util/* @DataDog/dd-trace-js
packages/dd-trace/test/plugins/util/test.spec.js @DataDog/ci-app-libraries
```

---

it doesn't matter. I'd say let's keep them as is, as it's bound to break anyway